xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -195,9 +196,6 @@ ADDON_STATUS CPVRClient::Create(int iClientId)
   catch (std::exception &e) { LogException(e, __FUNCTION__); }
 
   m_bReadyToUse = bReadyToUse;
-
-  SetEPGTimeFrame(CSettings::GetInstance().GetInt(CSettings::SETTING_EPG_DAYSTODISPLAY));
```

Comment:
If it becomes not used anymore can be the function `PVR_ERROR SetEPGTimeFrame(int iDays);` also removed, need the only a MIN_API_VERSION increase.